poetry.lock

```diff
@@ -161,11 +161,11 @@ toml = "*"
 
 [[package]]
 name = "babel"
-version = "2.9.1"
+version = "2.10.1"
```

Comment:
Please don't bump versions of not relevant modules in this PR.

Reply:
How do you manage not to modify this one, when you do `poetry update` after adding a library? That's not the right way to do it?

Reply:
You can not commit all lines but just lines which you need (`shotgun-api3` in this case). Most of these version changes breaks compatibility for older pythons.

Reply:
Created PR https://github.com/Ellipsanime/OpenPype/pull/6

Reply:
Done

Reply:
I'm really really sorry, I used old poetry.lock as reference, now they're older then should be (a lot).... I'll create new PR

Reply:
I will do it!

Reply:
it's good now.

Reply:
last change https://github.com/Ellipsanime/OpenPype/pull/7